webapp/sources/api-doc/openapi.src.yml

```diff
@@ -153,6 +151,8 @@ paths:
     $ref: paths/system/archives-restore.yml
   "/system/archives/{archiveKind}/zip/{commitId}":
     $ref: paths/system/archives-zip.yml
+  "/system/healthcheck":
+    $ref: paths/system/healthcheck.yml
```

Comment:
You also need to remove the "Health check" tag from the openapi.src.yml and healthcheck.yml files